nixos/modules/services/networking/wireguard.nix

```diff
@@ -160,6 +160,14 @@ let
         interval of 25 seconds; however, most users will not need this.'';
       };
 
+      table = mkOption {
+        default = "main";
+        type = types.str;
+        description = ''The kernel routing table to add this peer's associated
```

Comment:
Right, I forgot about the case where one configures one interface with many peers and wants their routing table entries in one table *that is not `main`*.

Do you think that's going to be a common requirement? It shouldn't be too hard to also introduce a per-interface option that sets a default for the per-peer options.

> This can't happen. Watch what happens when you try:

Okay, this seems to mean that for primary/primary, or even primary/secondary setups, using multiple interfaces is necessary.

I'm fine with this! Basically, you introduced a complication in WireGuard's design to make a pretty common case easier to set up. :)